packages/react-components/src/shared/src/styles.ts

```diff
@@ -1,24 +1,24 @@
 import { createContext, useContext } from "react";
 import { isNil } from "lodash";
 
-export const StyleContext = createContext();
+export const StyleContext = createContext<Record<string, any>>(null);
```

Comment:
null is not the same as undefined which was the original value.

Reply:
good point. 

also, FYI : https://github.com/DefinitelyTyped/DefinitelyTyped/pull/24509#issuecomment-382213106

undefined might be a weird default value